pkg/validator/types.go

```diff
@@ -197,3 +204,34 @@ type ResultMessage struct {
 	Type     MessageType
 	Category string
 }
+
+// ReadAuditFromFile reads the data from a past audit stored in a JSON or YAML file.
+func ReadAuditFromFile(fileName string) AuditData {
+	auditData := AuditData{}
+	oldFileBytes, err := ioutil.ReadFile(fileName)
```

Comment:
Is there a need here to use ioutil.ReadFile to first read the bytes into memory before running `ParseAudit`? Is this split up for testing purposes or could this be made more concise by using `os.Open(fileName)` and passing that to the `NewYAMLOrJSONDecoder()` function? `os.Open()` Returns a `File` type which conforms to the `io.Reader` interface.